tests/core/conversion/converters/test_element_wise.cpp

```diff
@@ -241,7 +241,7 @@ TEST(Converters, ATenRsubWithScalarConvertsCorrectly) {
 TEST(Converters, ATenClampMinConvertsCorrectly) {
   const auto graph = R"IR(
   graph(%x.1 : Tensor):
-          %2 : int = prim::Constant[value=-2]()
+          %2 : int = prim::Constant[value=1.5]()
```

Comment:
should this be float or int?

Reply:
input range is [1,5), in that case, -2 is never used

Reply:
sure but the output type should be float if the value is 1.5 right?

Reply:
updated